integration-tests/rbac-permission.test.ts

```diff
@@ -14,16 +14,16 @@ test('practitioner role can create new patient', async () => {
 
 describe('Negative tests', () => {
     test('invalid token', async () => {
-        const client = await getFhirClient('practitioner', 'Invalid token');
+        const client = await getFhirClient({ role: 'practitioner', providedAccessToken: 'Invalid token' });
         await expect(client.post('Patient', randomPatient())).rejects.toMatchObject({
             response: { status: 401 },
         });
     });
 
     test('auditor role cannot create new patient record', async () => {
-        const client = await getFhirClient('auditor');
+        const client = await getFhirClient({ role: 'auditor' });
         await expect(client.post('Patient', randomPatient())).rejects.toMatchObject({
-            response: { status: 403 },
+            response: { status: 401 },
```

Comment:
This is needed because of https://github.com/awslabs/fhir-works-on-aws-routing/pull/92

That routing change was released but it has not been picked up by the deployment package yet.